cuqi/experimental/mcmc/_laplace_approximation.py

```diff
@@ -0,0 +1,150 @@
+import scipy as sp
+import numpy as np
+import cuqi
+from cuqi.solver import CGLS
+from cuqi.experimental.mcmc import SamplerNew
+
+class UGLANew(SamplerNew):
+    """ Unadjusted (Gaussian) Laplace Approximation sampler
+    
+    Samples an approximate posterior where the prior is approximated
+    by a Gaussian distribution. The likelihood must be Gaussian.
+
+    Currently only works for LMRF priors.
+
+    The inner solver is Conjugate Gradient Least Squares (CGLS) solver.
+
+    For more details see: Uribe, Felipe, et al. "A hybrid Gibbs sampler for edge-preserving 
+    tomographic reconstruction with uncertain view angles." arXiv preprint arXiv:2104.06919 (2021).
```

Comment:
Update ref to published version: https://doi.org/10.1137/21M1412268

Reply:
Done